website/pages/docs/typography/heading.mdx

```diff
@@ -111,6 +111,4 @@ prop. No need to write `css` or `styled()`
 The Heading composes the [Box](/docs/layout/box) component, so you can pass all
 `Box` props.
 
-| Name   | Type                                               | Default | Description              |
-| ------ | -------------------------------------------------- | ------- | ------------------------ |
-| `size` | `4xl`, `3xl`, `2xl`, `xl` , `lg`, `md`, `sm`, `xs` | `xl`    | The size of the heading. |
+<PropsTable of="Heading" omit={null} />
```

Comment:
Unnecessary `null` here?

Reply:
`null` is intended to override the default value:

https://github.com/chakra-ui/chakra-ui/blob/439d992aa5ce7d1508a63cc041beb52bff70a3d4/website/src/components/props-table.tsx#L25

An empty array would be suitable as well.

Reply:
Ohhhh, yeap, that makes sense! If that were more common I'd say it makes more sense to have like a separate `hasTextProps` prop or something, but since it's only `Text` and `Heading` that should ever have that, I think this is totally fine :)